From 8401198559ec15879907d594291e28a95da336a8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Daniel=20D=C3=ADaz?= <daniel.diaz@linaro.org>
Date: Fri, 1 Feb 2019 11:03:57 -0600
Subject: [PATCH] Define SIGRTMIN and SIGRTMAX
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Daniel Díaz <daniel.diaz@linaro.org>
---
 include/old/test.h | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/include/old/test.h b/include/old/test.h
index 0738237e9..a34d13a2e 100644
--- a/include/old/test.h
+++ b/include/old/test.h
@@ -50,6 +50,20 @@
 #define NUMSIGS NSIG
 #endif
 
+#ifndef SIGRTMIN
+# define SIGRTMIN (__libc_current_sigrtmin ())
+#endif
+
+#ifndef SIGRTMAX
+# define SIGRTMAX (__libc_current_sigrtmax ())
+#endif
+
+#ifndef __SIGRTMIN
+# define __SIGRTMIN 32
+#endif
+#ifndef __SIGRTMAX
+# define __SIGRTMAX (_NSIG - 1)
+#endif
 
 /* defines for unexpected signal setup routine (set_usig.c) */
 #define FORK    1		/* SIGCHLD is to be ignored */
-- 
2.17.1

